crates/ark/src/help_proxy.rs

```diff
@@ -242,3 +253,22 @@ async fn preview_img(params: web::Query<PreviewRdParams>) -> HttpResponse {
 
     HttpResponse::Ok().content_type(mime_str).body(content)
 }
+
+// Conversion helper between reqwest and actix-web's `HeaderValue`
+//
+// Both point to a re-exported `HeaderValue` from the http crate, but they come from
+// different versions of the http crate, so they look different to Rust and we need a
+// small conversion helper.
+// - reqwest re-exports http 1.0.0's `HeaderValue`
+// - actix-web re-exports http 0.2.7's `HeaderValue`
+fn convert_header_value(x: &reqwest::header::HeaderValue) -> actix_web::http::header::HeaderValue {
+    let out = actix_web::http::header::HeaderValue::from_bytes(x.as_bytes());
+
+    // We've checked these are the same underlying structure, so assert that this works.
+    let mut out = out.unwrap();
+
+    // Set the one other field that defaults to `false`, just in case.
+    out.set_sensitive(x.is_sensitive());
+
+    out
+}
```

Comment:
Joint work with @DavisVaughan who discovered that:

- actix-web re-exports http 0.2.7's `HeaderValue`
- reqwest re-exports http 1.0.0's `HeaderValue`

Here is the commit where reqwest changed this:
https://github.com/seanmonstar/reqwest/commit/eb94f26919e881177ad3d9cd172f47d1a8263799